README.md

```diff
@@ -10,22 +10,20 @@
 
 ## **Description of the project:**
 
-<b>The Next Gen AI Powered Todo-List App.</b>
```

Comment:
I think this should stay.
it's way better when abbreviated.

Reply:
Changes done.Thanks.